pkgs/by-name/pi/piliplus/package.nix

```diff
@@ -0,0 +1,107 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter335,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  git,
+  mpv-unwrapped,
+  libplacebo,
+}:
+
+let
+  version = "1.1.4.1";
+  description = "Third-party Bilibili client developed in Flutter";
+in
+flutter335.buildFlutterApplication {
+  pname = "piliplus";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "bggRGjQaUbCoE";
+    repo = "PiliPlus";
+    tag = version;
+    hash = "sha256-sJKYfRoNmArHBsaHN9pARH9G6RrkRnRXDd28g2SezeU=";
+    leaveDotGit = true;
+  };
+
+  postPatch = ''
+    chmod +x lib/scripts/build.sh
+    patchShebangs lib/scripts/build.sh
+    substituteInPlace lib/scripts/build.sh \
+      --replace-fail "date" 'date -ud "@$SOURCE_DATE_EPOCH"'
+
+    # Disable auto-update check
+    substituteInPlace lib/utils/update.dart \
+      --replace-fail "if (kDebugMode) " ""
+  '';
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+  gitHashes = {
+    auto_orientation = "sha256-0QOEW8+0PpBIELmzilZ8+z7ozNRxKgI0BzuBS8c1Fng=";
+    canvas_danmaku = "sha256-3cBsQCvXuc5XvSRNY4QX33+t8aF2AfOQhRt+MCQWdOA=";
+    chat_bottom_container = "sha256-um9KwZUDxWBhFsGHfv00TjPzxDHmp43TLRF0GwuV1xs=";
+    extended_nested_scroll_view = "sha256-5X8ghUlEO/lvz/3PmYuipCjcs+QrIciaH5wgWp9i+24=";
+    floating = "sha256-TJ2i3hTOQ4euHWr+lfQU44L3vVehmWVdZuAzNyVaEfA=";
+    flutter_sortable_wrap = "sha256-Qj9Lzh+pJy+vHznGt5M3xwoJtaVtt00fxm4JJXL4bFI=";
+    material_design_icons_flutter = "sha256-KMwjnzJJj8nemCqUCSwYafPOwTYbtoHNenxstocJtz4=";
+    media_kit_video = "sha256-wtxTbqOkwQvgDX+LBWLZAdgKkTj4LC484if1LOMjbmA=";
+    webdav_client = "sha256-euNF7HdDtZ68BqSEq9BvO10BK09MxX2wWGoElFS0yeE=";
+  };
+
+  nativeBuildInputs = [
+    git # used in lib/scripts/build.sh to get the current commit hash
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  buildInputs = [
+    mpv-unwrapped
+  ]
+  ++ mpv-unwrapped.buildInputs
+  ++ libplacebo.buildInputs;
```

Comment:
Why?

Reply:
That's a concerning amount to be missing.

https://github.com/NixOS/nixpkgs/blob/dcf1d0bddebf420c8482ffb6a63ee5d0a1a3c5f1/pkgs/by-name/li/libplacebo/package.nix#L51-L66
https://github.com/NixOS/nixpkgs/blob/5a0711127cd8b916c3d3128f473388c8c79df0da/pkgs/applications/video/mpv/default.nix#L166-L226